src/main/java/ru/yandex/practicum/filmorate/controllers/FilmController.java

```diff
@@ -0,0 +1,50 @@
+package ru.yandex.practicum.filmorate.controllers;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import ru.yandex.practicum.filmorate.exceptions.NoSuchFilm;
+import ru.yandex.practicum.filmorate.model.Film;
+import javax.validation.Valid;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/films")
+@Slf4j
+public class FilmController {
+    private int idCounter = 0;
+    private final Map<Integer, Film> films = new HashMap<>();
+
+    @GetMapping
+    public List<Film> findAll() {
+        log.info("The list of films returned");
+        List<Film> result = new ArrayList<>(films.values());
```

Comment:
Так как переменная больше нигде не используется, лучше сразу возвращать список:
```suggestion
        return new ArrayList<>(films.values());
```